build.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const chokidar = require('chokidar');
+const esbuild = require('esbuild');
+const glob = require('tiny-glob');
+const serveHandler = require('serve-handler');
+const http = require('http');
+
+const args = process.argv;
+const PORT = 3001;
+
+async function build(opts = {}) {
+  const start = Date.now();
+
+  await esbuild.build({
+    entryPoints: ['./index.js'],
+    bundle: true,
```

Comment:
Can we minify here too?

Reply:
Done!

Reply:
This doesn't appear to be minifying for me. What changed?

Reply:
Yeah, that was it. At some point we're going to have to figure out how to minify the templates too (not today).